libs/jump_portrait/src/jump_portrait/utils.py

```diff
@@ -82,13 +85,34 @@ def parallel(
 
 def batch_processing(f: Callable):
     # This assumes parameters are packed in a tuple
-    def batched_fn(item_list: Iterable, job_idx: int, *args, **kwargs):
+    def batched_fn(item_list: Iterable, job_idx: int,
+                   print_progress:bool=True,
+                   *args, **kwargs):
         results = []
-        for item in item_list:
-            # pbar.set_description(f"Processing {item}")
+        for item in tqdm(item_list, position=0, leave=True,
+                         disable=not print_progress,
```

Comment:
I think 'leave' may cause troubles. We should test it on the command line, by running it in a script, and alongside notebooks (which it was not supporting originally).

Reply:
I will try both and let you know. From what I remember it allows to work both on notebook and script.